content/javascript/concepts/statements/statements.md

```diff
@@ -0,0 +1,124 @@
+---
+Title: 'Statements'
+Description: 'In JavaScript, a statement is a unit of code that performs a specific action or task.'
+Subjects:
+  - 'Web Development'
+  - 'Computer Science'
+Tags:
+  - 'Statements'
+  - 'Syntax'
+CatalogContent:
+  - 'introduction-to-javascript'
+  - 'paths/front-end-engineer-career-path'
+---
+
+In JavaScript, **statements** are instructions that are executed by the browser or Node.js. A statement can be a simple or complex operation that performs an action, such as assigning a value to a variable, calling a function, or controlling program flow with conditional statements.
+
+
+## Expression Statements
+
+Expression statements, such as function calls or assignments, evaluate an expression and generate a value in JavaScript. They can be assigned to a [variable](https://www.codecademy.com/resources/docs/javascript/variables) or used as part of a larger expression.
+
+### Syntax
+
+```js
+// Function call
+console.log('Hello World');
+// Assignments
+// Assign 5 to x
+var x = 5;
+// Assign 6 to X
+var X = 6;
+```
+
+In this statement, a `const` variable named `greeting` is declared and assigned the value of `'Hello World'`:
+
+### Syntax
+
+```js
+const greeting = 'Hello World';
+```
+
+The first statement below assigns the value 5 to the variable `x`, while the second statement logs the value of `x` to the console:
+
+### Syntax
+
+```js
+let x = 5;
+console.log(x);
+```
+
+JavaScript [variables](https://www.codecademy.com/resources/docs/javascript/variables) are containers for storing data values.
+In this example, `x`, `y`, `X`, and `z`, are [variables](https://www.codecademy.com/resources/docs/javascript/variables), declared with the `var` keyword.
+JavaScript identifiers are also case-sensitive. Note that `x` is not the same as `X`:
+
+### Syntax
+
+```js
+var x = 5;
+var y = 6;
+var X = 4;
+var z = x + y;
+```
```

Comment:
@nadocodes Of course! What's most important is understanding and filling in any gaps or misunderstandings. Which is a great outlook because I made an error here.

I tried putting in a suggestion but there's some formatting issues which won't let me put it in as a suggestion from lines 18-62 because of the early suggestion block's closing backticks. This is what I recommend:

```
## Declaration Statements

Declaration statements are variables, function, or classes that are introduced into a program. These statements begin with a keyword followed by its identifier or name.

```js
var x = 5;
var y = 6;
var X = 4;
var z = x + y;
```

JavaScript [variables](https://www.codecademy.com/resources/docs/javascript/variables) are containers for storing data values. In this example, `x`, `y`, `X`, and `z`, are declared with the `var` keyword.

JavaScript identifiers are also case-sensitive (Note that `x` is not the same as `X`).

## Expression Statements

Expression statements, such as function calls or assignments, evaluate an expression and generate a value in JavaScript. They can be assigned to a variable or used as part of a larger expression and are then discarded.

```js
// Function call
console.log('Hello World');
```

The second statement below re-assigns the value 5 to the variable `x`, while the third statement logs the value of `x` to the console:

```js
var x = 5;
x = 8;
console.log(x);
```
```

Reply:
Our original understanding of declaration statements v expression statements was in fact technically accurate. The Mozilla docs tripped me up

Reply:
Perfect!! I have made the changes and hoping I havnt missed anything, I am wondering if I should leave the "const greeting = 'Hello World'; to highlight varied variable use?

Reply:
Thank you 😄